src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReportService;
+import core.basesyntax.service.StorageService;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class ReportServiceImpl implements ReportService {
+    private final StorageService storageService;
+
+    public ReportServiceImpl() {
+        storageService = new StorageServiceImpl();
+    }
```

Comment:
I think it's better to pass StorageService as a constructor parameter, but I'm not sure, let mentor take a look.

Reply:
done 👌🏻